resources/puppetlabs/puppetdb/pql/pql-grammar-experimental.ebnf

```diff
@@ -30,7 +28,7 @@ extract = <lbracket>, [<whitespace>], [extractfields], [<whitespace>], <rbracket
 <extractfields> = fieldlist;
 
 (* Filtering *)
-<where> = <lbrace>, [<whitespace>], [expression], [<whitespace>], [modifier], [<whitespace>], <rbrace>;
+<where> = <lbrace>, [<whitespace>], [expression], [<whitespace>], {modifier | pagingclause, [<whitespace>]}, [<whitespace>], <rbrace>;
```

Comment:
It's an organisational nitpick but I lumped pagination with grouping all as 'modifiers' logically in my head, but with the group by potential change I guess this goes away. If we think we're wanting to separate this into two places, perhaps modifier should just become group by directly and we collapse one layer.